lib/wallaby/experimental/selenium.ex

```diff
@@ -0,0 +1,158 @@
+defmodule Wallaby.Experimental.Selenium do
+  @moduledoc false
+
+  @behaviour Wallaby.Driver
+
+  alias Wallaby.{Element, Session}
+  alias Wallaby.Experimental.Selenium.WebdriverClient
+
+  @type start_session_opts ::
+    {:remote_url, String.t} |
+    {:capabilities, map} |
+    {:create_session_fn, ((String.t, map) -> {:ok, %{}})}
+
+  @spec start_session([start_session_opts]) :: {:ok, Session.t}
+  def start_session(opts \\ []) do
+    base_url = Keyword.get(opts, :remote_url, "http://localhost:4444/wd/hub/")
```

Comment:
You'd start a session by calling:
```elixir
Wallaby.start_session(driver: Wallaby.Experimental.Selenium, remote_url: "http://localhost:4444/wd/hub")
```

The `:driver` option determines which module to call `start_session/1` on and the rest of the options are passed through directly to the driver. That way chromedriver could look like this:

```elixir
Wallaby.start_session(driver: Wallaby.Experimental.ChromeDriver, remote_url: "http://localhost:4444/wd/hub")
```